AutoIncrementIDsScript.sql

```diff
@@ -1,35 +1,18 @@
-Please run the following queries in the specified order in case of first time deployment:
-
+use motech_data_services;
 SET FOREIGN_KEY_CHECKS=0;
-
 ALTER TABLE `nms_districts` CHANGE COLUMN `id` `id` BIGINT(20) NOT NULL AUTO_INCREMENT ;
-
 ALTER TABLE `nms_talukas` CHANGE COLUMN `id` `id` BIGINT(20) NOT NULL AUTO_INCREMENT ;
-
 ALTER TABLE `nms_health_blocks` CHANGE COLUMN `id` `id` BIGINT(20) NOT NULL AUTO_INCREMENT;
-
 ALTER TABLE `nms_health_facilities` CHANGE COLUMN `id` `id` BIGINT(20) NOT NULL AUTO_INCREMENT;
-
 ALTER TABLE `nms_health_sub_facilities` CHANGE COLUMN `id` `id` BIGINT(20) NOT NULL AUTO_INCREMENT;
-
 ALTER TABLE `nms_villages` CHANGE COLUMN `id` `id` BIGINT(20) NOT NULL AUTO_INCREMENT;
-
 ALTER TABLE `nms_mother_rejects` CHANGE COLUMN `id` `id` BIGINT(20) NOT NULL AUTO_INCREMENT;
-
 ALTER TABLE `nms_child_rejects` CHANGE COLUMN `id` `id` BIGINT(20) NOT NULL AUTO_INCREMENT;
-
 ALTER TABLE `nms_imi_cdrs` CHANGE COLUMN `id` `id` BIGINT(20) NOT NULL AUTO_INCREMENT ;
-
 ALTER TABLE `nms_imi_csrs` CHANGE COLUMN `id` `id` BIGINT(20) NOT NULL AUTO_INCREMENT ;
-
-ALTER TABLE `motech_data_services`.`nms_taluka_healthblock`
-ADD COLUMN `creationDate` DATETIME NULL DEFAULT NULL AFTER `healthblock_id`,
-ADD COLUMN `modificationDate` DATETIME NULL DEFAULT NULL AFTER `creationDate`;
-
-ALTER TABLE `motech_data_services`.`nms_village_healthsubfacility`
-ADD COLUMN `creationDate` DATETIME NULL DEFAULT NULL AFTER `village_id`,
-ADD COLUMN `modificationDate` DATETIME NULL DEFAULT NULL AFTER `creationDate`;
-
-
+ALTER TABLE `motech_data_services`.`nms_taluka_healthblock`;
+CHANGE COLUMN `id` `id` BIGINT(20) NOT NULL AUTO_INCREMENT;
```

Comment:
Is this valid SQL? Looks like you cut the original statement in half with a semicolon(`;`)

Reply:
No, this is a file used for reference -- not a valid SQL. changed it and pushed